charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.31.1
-version: 15.0.4
+version: 15.0.5
```

```suggestion
version: 15.1.1
```

Please update chart version without creating a new PR. You can just rebase.

---

I attempted to update this version but now it says I have a conflict in the PR... Sorry I'm not very experienced with github.